scripts/calibration/jaiabot-calibration.py

```diff
@@ -154,9 +153,9 @@ def skipWing():
     output = rd()
     output = str(output)
     output = output.split()
-    print("current upper bound is", output[0])
-    print("current lower bound is", output[1])
-    print("current center bound is", output[2])
+    print("current upper bound is 1900")
```

Comment:
Should we be using magic numbers for these print outs? Is there a variable we can put here instead?